package.json

```diff
@@ -32,6 +32,7 @@
   "dependencies": {
     "@asyncapi/generator-filters": "^1.0.0",
     "@asyncapi/generator-hooks": "^0.1.0",
+    "@asyncapi/generator-model-sdk": "file:../generator-model-sdk",
```

Comment:
Must be changed 😄

Reply:
I use some presets from this PR https://github.com/asyncapi/generator-model-sdk/pull/112 and I have symlink here to use them :) Probably I will move presets to this repo, but don't worry, I remember about it.

Reply:
Get used to `npm link`. This way you don't have to change things here: https://docs.npmjs.com/cli/v6/commands/npm-link.